include/seqan3/core/simd/detail/builtin_simd.hpp

```diff
@@ -97,11 +97,11 @@ struct builtin_simd_traits_helper<builtin_simd_t>
     //!\brief The scalar type of builtin_simd_t
     using scalar_type = std::remove_reference_t<decltype(std::declval<builtin_simd_t>()[0])>;
     //!\brief The length of builtin_simd_t
-    static constexpr auto length = min_viable_uint_v<sizeof(builtin_simd_t) / sizeof(scalar_type)>;
+    static constexpr auto length = sizeof(builtin_simd_t) / sizeof(scalar_type);
```

Comment:
Are you sure this has no side effects?
`sizeof` returns `size_t`, so the type of `length` was probably `uint8_t` before, but now it is `size_t`. I don't know if this may have any implications here.

Reply:
Since it is constexpr, this doesn't really have an effect. The only effect was that I could remove another header.

Reply:
Ok, so it was just the fad of casting it into the smallest possible type and not a carefully conducted optimization.